src/pk_vm.c

```diff
@@ -1234,15 +1234,28 @@ static PkResult runFiber(PKVM* vm, Fiber* fiber) {
     {
       // Don't pop yet, we need the reference for gc.
       Var r = PEEK(-1), l = PEEK(-2);
-      Var value = varBitAnd(vm, l, r);
+      Var result = varBitAnd(vm, l, r);
       DROP(); DROP(); // r, l
-      PUSH(value);
+      PUSH(result);
 
       CHECK_ERROR();
       DISPATCH();
     }
 
     OPCODE(BIT_OR):
+    {
+      // Don't pop yet, we need the reference for gc.
+      Var r = PEEK(-1), l = PEEK(-2);
+
+      // Result is in base 10 (decimal)
```

Comment:
The decimal or binary is a representation of the underlying number, and the result doesn't have a base value, we currently have [`hex()`](https://github.com/ThakeeNathees/pocketlang/blob/473daa4716bcccf58add33d8f952665a90bf9411/tests/lang/core.pk#L3) function which can return the hexadecimal representation of the a number, `bin` and binary/hex literals (`0b101001`, `0x12abc5f`) are [TODO](https://github.com/ThakeeNathees/pocketlang/blob/473daa4716bcccf58add33d8f952665a90bf9411/docs/TODO.txt#L11).

This comment can be removed.